operate/qa/integration-tests/src/test/java/io/camunda/operate/it/ModifyProcessInstanceOperationZeebeIT.java

```diff
@@ -149,7 +148,6 @@ public void shouldAddTokenWithVariables() throws Exception {
         .isEqualTo("\"d\"");
   }
 
-  @Ignore("Due to flaky CI tests")
```

Comment:
this had been disabled since 2023. We can keep an eye on this, but I would rather invest in fixing/removing it rather than have it stick around and not running